src/BaseDAO/ShareTest/FA2.hs

```diff
@@ -163,7 +163,7 @@ updatingOperatorScenario originateFn = do
   callFrom (AddressResolved owner2) dao (Call @"Transfer") transferParams
     & expectCustomError_ #fA2_NOT_OPERATOR
   callFrom (AddressResolved owner1) dao (Call @"Update_operators") ([FA2.AddOperator notOwnerParams])
-    & expectCustomError_ #nOT_OWNER
+    & expectCustomError_ #fA2_NOT_OPERATOR
```

Comment:
Not sure why this was changed or how it relates to the problems on CI, but it seems the tests are now working and this is the only thing failing (correctly, as it should indeed be `#nOT_OWNER` AFAIK).
```suggestion
    & expectCustomError_ #nOT_OWNER
```